src/crypto/password/pbkdf2.clj

```diff
@@ -30,12 +38,15 @@
   ([raw iterations]
      (encrypt raw iterations (random/bytes 8)))
   ([raw iterations salt]
+     (encrypt raw iterations salt "HMAC-SHA1"))
+  ([raw iterations salt algorithm]
      (let [key-length  160
            key-spec    (PBEKeySpec. (.toCharArray raw) salt iterations key-length)
-           key-factory (SecretKeyFactory/getInstance "PBKDF2WithHmacSHA1")]
+           key-factory (SecretKeyFactory/getInstance (code-to-algorithm algorithm))]
        (->> (.generateSecret key-factory key-spec)
             (.getEncoded)
             (encode-str)
+            (str (encode-str (.getBytes algorithm)) "$")
```

Comment:
This could just be `(str algorithm "$")`. There's no need to encode the string, because we know the strings have no dangerous bytes.